components/Code/Code.tsx

```diff
@@ -14,6 +15,14 @@ interface CodeProps {
 
 function Code({code, language}: CodeProps){
 
+  let [fontsLoaded] = useFonts({
+    Hack: require("../../assets/fonts/Hack-Regular.ttf"),
+  });
+
+  const courierFont = Platform.Version >= 15 ? 'Courier New' : 'Courier';
```

Comment:
I just tested on iOS. It seems like Platform.Version returns a string on Android and a number on iOS, which typescript complains about. And we can get the major version number from Platform.Version:
` const courierFont = (Platform.OS === "ios" && parseInt(Platform.Version, 10)>=15) ? 'Courier New' : 'Courier';`